decidim-budgets/config/locales/en.yml

```diff
@@ -376,6 +376,26 @@ en:
           email_outro: You have received this notification because you are following %{participatory_space_title}. You can stop receiving notifications following the previous link.
           email_subject: The %{resource_title} budget is now active for %{participatory_space_title}.
           notification_title: The <a href="%{resource_path}">%{resource_title}</a> budget is now active for %{participatory_space_title}.
+    open_data:
+      help:
+        budgets:
```

Comment:
while testing, i have found this as well, that i did not see first time. 

```
* id: Translation missing: en.decidim.open_data.help.projects.id
* category: Translation missing: en.decidim.open_data.help.projects.category
* scope: Translation missing: en.decidim.open_data.help.projects.scope
* participatory_space: Translation missing: en.decidim.open_data.help.projects.participatory_space
* component: Translation missing: en.decidim.open_data.help.projects.component
* title: Translation missing: en.decidim.open_data.help.projects.title
* description: Translation missing: en.decidim.open_data.help.projects.description
* budget: Translation missing: en.decidim.open_data.help.projects.budget
* budget_amount: Translation missing: en.decidim.open_data.help.projects.budget_amount
* confirmed_votes: Translation missing: en.decidim.open_data.help.projects.confirmed_votes
* comments: Translation missing: en.decidim.open_data.help.projects.comments
* created_at: Translation missing: en.decidim.open_data.help.projects.created_at
* url: Translation missing: en.decidim.open_data.help.projects.url
* address: Translation missing: en.decidim.open_data.help.projects.address
* related_proposals: Translation missing: en.decidim.open_data.help.projects.related_proposals
* related_proposal_titles: Translation missing: en.decidim.open_data.help.projects.related_proposal_titles
* related_proposal_urls: Translation missing: en.decidim.open_data.help.projects.related_proposal_urls
```

Reply:
You didn't see it because it wasn't there :) 

This is related to [2912e9f](https://github.com/decidim/decidim/pull/13435/commits/2912e9f88fc2c099ad93fd4886b6b6b5299c5dff), as we're changing the key for the i18n, from the module name to the model/resource name. 

One thing that's bothering me is that this particular bug should've been detected by the specs.

```
bin/rspec decidim-budgets/spec/services/decidim/open_data_exporter_spec.rb
```

Locally I see that's failing, but in the CI is passing.

Can you confirm if it's failing in your machine too?

Reply:
Also, there's another failing spec that you've been detected:
```
bin/rspec decidim-core/spec/services/decidim/open_data_exporter_spec.rb
``` 

Not sure what is happening

Reply:
They both fail on my local environment ...